tests/snuba/search/test_backend.py

```diff
@@ -2567,25 +2566,18 @@ class EventsSnubaSearchTest(TestCase, EventsSnubaSearchTestCases):
 @apply_feature_flag_on_cls("organizations:issue-search-group-attributes-side-query")
 class EventsJoinedGroupAttributesSnubaSearchTest(TransactionTestCase, EventsSnubaSearchTestCases):
     def setUp(self):
-        def post_insert(snapshot: GroupAttributesSnapshot):
+        def post_insert(snapshot: GroupAttributesSnapshot) -> None:
             from sentry.utils import snuba
 
-            try:
-                resp = snuba._snuba_pool.urlopen(
-                    "POST",
-                    "/tests/entities/group_attributes/insert",
-                    body=json.dumps([snapshot]),
-                    headers={},
-                )
-                if resp.status != 200:
-                    raise snuba.SnubaError(
-                        f"HTTP {resp.status} response from Snuba! {json.loads(resp.data)}"
-                    )
```

Comment:
Not totally sure I understand this change. It seems like we're losing out on a detailed message here

Reply:
even if it were useful it would never be hit -- urllib3 raise_for_status already happened

Reply:
Is `raise_for_status` called inside of `urlopen`? I don't see it being called otherwise here.

Reply:
yeah